proctl/variables.go

```diff
@@ -897,6 +934,28 @@ func (thread *ThreadContext) FunctionArguments() ([]*Variable, error) {
 	return thread.variablesByTag(dwarf.TagFormalParameter)
 }
 
+// PackageVariables returns the name, value, and type of all package variables in the application
```

Comment:
Could you add a period to the end of the comment, also I tend to like to group exported methods together, so maybe move this method as well as `LocalVariables` and `FunctionArguments` up a bit in the file. (Again, nit-picky I know but I feel it keeps things a bit more organized).